charts/jaeger/templates/ingester-hpa.yaml

```diff
@@ -18,10 +18,26 @@ spec:
     - type: Resource
       resource:
         name: memory
+        {{- if eq "autoscaling/v2" (include "common.capabilities.hpa.apiVersion" ( dict "context" $ )) }}
+        target:
+          averageUtilization: {{ .Values.ingester.autoscaling.targetMemoryUtilizationPercentage }}
+          type: Utilization
+        {{- else }}            
         targetAverageUtilization: {{ .Values.ingester.autoscaling.targetMemoryUtilizationPercentage }}
+        {{- end }} 
   {{- end }}
     - type: Resource
       resource:
         name: cpu
+        {{- if eq "autoscaling/v2" (include "common.capabilities.hpa.apiVersion" ( dict "context" $ )) }}        
+        target:
+          averageUtilization: {{ .Values.ingester.autoscaling.targetCPUUtilizationPercentage | default 80 }}
+          type: Utilization
+        {{- else }}    
         targetAverageUtilization: {{ .Values.ingester.autoscaling.targetCPUUtilizationPercentage | default 80 }}
+        {{- end }}
+  {{- with .Values.ingester.autoscaling.behavior }}
+  behavior:
```

Comment:
let's add default value as {} in values.yaml file also.

Reply:
done